packages/node-sdk/README.md

```diff
@@ -0,0 +1,170 @@
+  boundClient.trackFeatureUsage({
+    event: "custom_login_used",
+    attributtes: { some: "attribute" },
+  });
+}
+```
+
+### Initialization Options
+
+Supply these to the `constructor` of the `Client` class:
+
+```ts
+{
+  // The secret key used to authenticate with the Bucket API.
+  secretKey: string,
+  // The host to send requests to (optional).
+  host?: string = "https://tracking.bucket.co",
+  // The interval to re-fetch feature flag definitions (optional).
+  // Default is 60 seconds.
+  refetchInterval?: number = 60000,
+  // The interval to re-fetch feature flags (optional). Default
+  // is 5 times the re-fetch interval.
+  staleWarningInterval?: number = 5 * 60000,
+  // The logger you can supply. By default no logging is performed.
+  logger?: Logger;
+  // The custom http client. By default the internal `fetchClient` is used.
+  httpClient?: HttpClient = fetchCient,
+}
+```
+
+### Zero PII
+
+The Bucket SDK doesn't collect any metadata and HTTP IP addresses are _not_ being
+stored. For tracking individual users, we recommend using something like database
+ID as userId, as it's unique and doesn't include any PII (personal identifiable
+information). If, however, you're using e.g. email address as userId, but prefer
+not to send any PII to Bucket, you can hash the sensitive data before sending
+it to Bucket:
+
+```ts
+import { sha256 } from 'crypto-hash';
+
+client.withUser({ userId: await sha256("john_doe"), ... });
+```
+
+### Custom Attributes & Context
+
+You can pass attributes as part of the object literal passed to the `withUser()`,
+`withCompany()` and `trackFeatureUsage()` methods. Attributes cannot be nested
+(multiple levels) and must be either strings, integers or booleans.
+
+Some attributes are used by Bucket.co and manged separately, such as:
+
+- `name` or `$name` -- display name for `user`/`company`,
+- `email` or `$email` -- the email of the user.
+
+You can supply an additional `context` to these functions as well. The `context`
+object contains additional data that Bucket uses to make some behavioural choices.
+
+By default, sending `updateCompany`, `updateUser` and `trackFeatureUsage` calls
+automatically update the given user/company `Last seen` property. You can control
+if `Last seen` should be updated when the events are sent by setting
+`context.active = false` to avoid updating last seen. This is often useful if you
+have a background job that goes through a set of companies just to update their.
+
+Example:
+
+```ts
+// create a new client initialized with an user and a company.
+const boundClient = client
+  .withUser({
+    userId: "188762",
+    attributes: { name: "John O." },
+    context: { active: true },
+  })
+  .withCompany({
+    companyId: "0083663",
+    attributes: { name: "My SaaS Inc." },
+    context: { active: false },
+  });
+
+// send updates to Bucket.co with the new details.
+await boundClient.updateCompany();
+await boundClient.updateUser();
```

Comment:
This seems like an unintuitive way of triggering the underlying user and company tracking events to bucket. You might well want to register different things in bucket than you'd want to provide as a feature flagging context.

Especially the `active: true/false` seems like something that is not relevant to feature flag checks at all, but it only relevant to updating entities in bucket. So I would expect the `active`-meta to be a part of the function that triggers these bucket HTTP API calls